en/advanced_config/README.md

```diff
@@ -1,32 +1,43 @@
 # Advanced Configuration
 
-This section contains configuration topics that are relevant when bringing up a new airframe (or significantly modifying a supported frame), and/or for OEM manufacturers.
-It covers vehicle-specific tuning, factory/OEM calibration, and more advanced/less-common configuration.
+This section contains contains more advanced configuration topics, and those for features that are more rarely modified, or that are specific only to manufacturers/OEMs.
 
-Note that the [peripheral hardware](../peripherals/README.md) documentation contains configuration that is specific to particular sensors and actuators.
+Parameter config:

```

Comment:
Note, I split this advanced config into sections, which leads me to the conclusion that they are not all that advanced - see rendered [here](https://github.com/PX4/PX4-user_guide/blob/9c18a38e722f155d281f85eabbb633f02549f2c3/en/advanced_config/README.md). They are "OEM specific" or "Config for features you might not touch, like serial ports or landing". 

I am wondering if it might be better to have a parent config, put standard config under that, then make advanced a peer (or split up and the parts are peers). Still thinking about that - depends a bit on whether I add features as an organisation structure.

Reply:
Maybe there are just mandatory and optional steps mapping to things you have to have/go through and setup of additional features someone might need for his vehicle or use case.

Reply:
Yes - that's what we do now with standard and advanced. That makes advanced very large and "can't see the wood for the trees". If you split it then perhaps you can provide more directed guidance - i.e. factory needs this, but most people don't. 

Right now I'm compromising by putting heading levels in the top level page http://docs.px4.io/main/en/advanced_config/ but the sections below are just a random set. 

I'm still thinking about it :-)